bing_bert/nvidia/modelingpreln.py

```diff
@@ -43,7 +43,7 @@
 import torch.nn.functional as F
 import torch.nn.init as init
 
-from deepspeed.ops.sparse_attention import SparseAttentionUtils
+#from deepspeed.ops.sparse_attention import SparseAttentionUtils

```

Comment:
/cc @arashashari can we move this import (any all client side SA imports) to only be done if the user wants to use sparse attention? Otherwise this will break our example code if the user is running on a system that does not support sparse attention. Ammar ran into this specific issue tonight since he was running torch 1.2 which is not supported by SA.

Reply:
I see your point. Sure, I will commit an update in few minutes.

Reply:
@awan-10 please close this if your tests were successful.